src/testType.ts

```diff
@@ -103,16 +103,17 @@ export class TestTypeImpl {
     suite._hooks.push({ type: name, fn, location: callLocation() });
   }
 
-  private _modifier(type: 'skip' | 'fail' | 'fixme' | 'slow', arg?: boolean | string | Function, description?: string) {
+  private _modifier(type: 'skip' | 'fail' | 'fixme' | 'slow', ...modiferAgs: [arg?: boolean | string | Function, description?: string]) {
```

Comment:
`arg` cannot be a string now, right?

Reply:
by definition any would probably be the right type, since it needs to be something truehy. wdyt?